index.html

```diff
@@ -68,7 +68,78 @@ <h3>Multi-Post Stories</h3>
                 <div class="item3">bootstrap</div>
                 <div class="item3">Ruby</div>   
             </div>
-            <button>Select Project</button>
+            <button type="button" class="work-button">Select Project</button>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads no accounts or sign ups required ha been the industry standups</p></div>
+                <ul class="grid-container2">
+                    <li class="item1"><a href="#">html</a></li>
+                    <li class="item2"><a href="#">bootstrap</a></li>
+                    <li class="item3"><a href="#">Ruby</a></li>  
+                </ul>
+                <button type="button" class="job-button">See Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <ul class="grid-container2">
+                    <li class="item1"><a href="#">html</a></li>
+                    <li class="item2"><a href="#">bootstrap</a></li>
+                    <li class="item3"><a href="#">Ruby</a></li>  
+                </ul>
+                <button type="button" class="job-button">See Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <ul class="grid-container2">
+                    <li class="item1"><a href="#">html</a></li>
+                    <li class="item2"><a href="#">bootstrap</a></li>
+                    <li class="item3"><a href="#">Ruby</a></li>  
+                </ul>
+                <button type="button" class="job-button">See Project</button>
+            </div>
+        </section>
+        <section>
+            <div class="me">
+                <div class="about-me">
+                    <h1>About me</h1>
```

Comment:
Kindly change the `<h1>` tag to any of `<h2>`, `<h3>`, ..., `<h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpages `<h1>` is the most important heading and should accurately summarize the contents of that page. You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).